iOS-TvShows/iOS-TvShows/Modules/Login/Router/LoginRouter.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  LoginRouter.swift
+//
+//  Created by Filip Gulan on 25/04/2019.
+//  Copyright © 2019 Infinum. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+class LoginRouter: Router {
+
+    static let baseUrl = "https://tv-shows.infinum.academy"
+    static let usersPath = "/users"
+
+    static func login(email: String, password: String) -> LoginRouter {
+        let params = [
+            "email": email,
+            "password": password
+        ]
+        return LoginRouter(
```

Comment:
If you open Router file there is commented out part of the code on the bottom - Consider moving baseUrl to constants and having convenience init which will always append same base url

Reply:
Great!